sim/adversary/decide.go

```diff
@@ -10,25 +10,39 @@ import (
 
 var _ Receiver = (*ImmediateDecide)(nil)
 
+type ImmediateDecideOption func(*ImmediateDecide)
```

Comment:
Use the term ` ImmediateDecision` instead of `ImmediateDecide`?

Reply:
Well, it's an option to `ImmediateDecide` where "decide" means "immediately send a decide message" not "immediately make a decision".

Reply:
And I missed that completely. Thanks Steven; nothing to see here.